app/controllers/api/reputation_controller.rb

```diff
@@ -3,7 +3,7 @@ class ReputationController < BaseController
     skip_before_action :ensure_onboarded!, only: [:index]
 
     def index
-      render json: AssembleReputationTokens.(current_user, params)
+      render json: AssembleReputationTokens.(current_user, list_params)
```

Comment:
And this wasn't calling the correct param, which meant for_header did work on reputation ironically.